app/views/users/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<html>
+  <body>
+    <h1>User Details</h1>
+  </body>
```

Comment:
- [ OPTIONAL ] I suggest this user index file should have placeholder "List of Users" since it is responsible for displaying the list of all users